### ISSUE TITLE
High max_bin for accuracy 

### ISSUE THREAD

**Issue by Mrforeman1** [microsoft/LightGBM#6016]
## Description
I'm currently comparing lightGBM with random forest and xgboost. In order for light GBM to have similar accuracy of  random Forest and Xgboost , I have to increase the max bin up to 5000. But from what I've seen online the max bin should be at a maximum of 255. I'm using around 100,000 rows for classification. The accuracy at 255 is around 0.5 but when I set it to 5,000 and jump to around .92.

**Comment by jameslamb** [comment]
Thanks for using LightGBM. What are you asking us to do?

**Comment by Mrforeman1** [comment]
I think I posted the statement in the wrong forum. I was looking to see if that was a realistic value

**Comment by jameslamb** [comment]
Ok, we'll close this then.

The particular set of hyperparameters that works well for a given problem depends on details you haven't provided like:

* the shape of the input data (number of features, distribution of the features, distribution of the target)
* the objective / loss function you're using
* other parameters that control model complexity, like `min_data_in_bin` and `num_iterations`

Use some hyperparameter-tuning strategy (like `lightgbm.cv()` in this project's Python package or `GridSearchCV` from `scikit-learn`) to find the set that works best for your data and problem.

It's not possible to say what a "realistic" expected fit to the training data should be given only the information you've provided.
